mappings/net/minecraft/test/StartupParameter.mapping

```diff
@@ -1,4 +1,239 @@
+		ARG 1 entity
+	METHOD method_35980 getBlockState (Lnet/minecraft/class_2338;)Lnet/minecraft/class_2680;
+		ARG 1 pos
+	METHOD method_35981 putAndRemoveRedstoneBlock (Lnet/minecraft/class_2338;J)V
+		ARG 1 pos
+		ARG 2 delay
+	METHOD method_35982 expectEntityWithData (Lnet/minecraft/class_2338;Lnet/minecraft/class_1299;Ljava/util/function/Function;Ljava/lang/Object;)V
+		ARG 1 pos
+		ARG 2 type
+		ARG 3 entityDatagGetter
+		ARG 4 data
+	METHOD method_35983 expectContainerWith (Lnet/minecraft/class_2338;Lnet/minecraft/class_1792;)V
+		ARG 1 pos
+		ARG 2 item
+	METHOD method_35984 setBlockState (Lnet/minecraft/class_2338;Lnet/minecraft/class_2248;)V
+		ARG 1 pos
+		ARG 2 block
+	METHOD method_35985 (Lnet/minecraft/class_2338;Lnet/minecraft/class_2248;Lnet/minecraft/class_2248;)Z
+		ARG 3 block1
+	METHOD method_35986 setBlockState (Lnet/minecraft/class_2338;Lnet/minecraft/class_2680;)V
+		ARG 1 pos
+		ARG 2 state
+	METHOD method_35987 expectBlockProperty (Lnet/minecraft/class_2338;Lnet/minecraft/class_2769;Ljava/lang/Comparable;)V
+		ARG 1 pos
+		ARG 2 property
+		ARG 3 value
+	METHOD method_35988 checkBlockProperty (Lnet/minecraft/class_2338;Lnet/minecraft/class_2769;Ljava/util/function/Predicate;Ljava/lang/String;)V
+		ARG 1 pos
+		ARG 2 property
+		ARG 3 predicate
+		ARG 4 errorMessage
+	METHOD method_35989 (Lnet/minecraft/class_2338;Lnet/minecraft/class_3341;Lnet/minecraft/class_2338;)V
+		ARG 3 checkedPos
+	METHOD method_35990 expectBlockWithStateOf (Lnet/minecraft/class_2338;Lnet/minecraft/class_2338;)V
+		ARG 1 checkedPos
+		ARG 2 correctStatePos
+	METHOD method_35991 checkBlock (Lnet/minecraft/class_2338;Ljava/util/function/Predicate;Ljava/lang/String;)V
+		ARG 1 pos
+		ARG 2 predicate
+		ARG 3 errorMessage
+	METHOD method_35992 checkBlock (Lnet/minecraft/class_2338;Ljava/util/function/Predicate;Ljava/util/function/Supplier;)V
+		ARG 1 pos
+		ARG 2 predicate
+		ARG 3 errorMessageSupplier
+	METHOD method_35993 addFinalTask (Ljava/lang/Runnable;)V
+		ARG 1 runnable
+	METHOD method_35999 (Ljava/util/function/Predicate;Lnet/minecraft/class_2680;)Z
+		ARG 1 state
+	METHOD method_36001 killAllEntities ()V
+	METHOD method_36002 toggleLever (III)V
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+	METHOD method_36003 waitAndRun (JLjava/lang/Runnable;)V
+		ARG 1 ticks
+		ARG 3 runnable
+	METHOD method_36004 dontExpectEntity (Lnet/minecraft/class_1299;)V
+		ARG 1 type
+	METHOD method_36005 dontExpectEntityToTouch (Lnet/minecraft/class_1299;DDD)V
+		ARG 1 type
+		ARG 2 x
+		ARG 4 y
+		ARG 6 z
+	METHOD method_36006 spawnMob (Lnet/minecraft/class_1299;FFF)Lnet/minecraft/class_1308;
+		ARG 1 type
+		ARG 2 x
+		ARG 3 y
+		ARG 4 z
+	METHOD method_36007 spawnMob (Lnet/minecraft/class_1299;III)Lnet/minecraft/class_1308;
+		ARG 1 type
+		ARG 2 x
+		ARG 3 y
+		ARG 4 z
+	METHOD method_36008 spawnMob (Lnet/minecraft/class_1299;Lnet/minecraft/class_243;)Lnet/minecraft/class_1308;
+		ARG 1 type
+		ARG 2 vec3d
```

Comment:
```suggestion
		ARG 2 pos
```

Reply:
I agree, better to use `pos` here because `vec3d` only describes the type (and "vector of 3 doubles" is not a useful parameter name). Why was this resolved with no change as well?